File: src/main/java/plus_minus/PlusMinus.java
package plus_minus;

import java.util.Random;

public class PlusMinus {
    public void plusMinus(){
        Random random = new Random();
        int[] arr = new int[5];
        int positive = 0;
        int negative = 0;
        int neutral = 0;

        for(int i =0; i<arr.length; i++){
            arr[i] = random.nextInt();
            System.out.println(java.util.Arrays.toString(arr));
            if(arr[i]>0){
                positive ++;
            }
            if(arr[i]<0){
                negative ++;
            }
            if(arr[i]==0){
                neutral ++;
            }
        }

        System.out.println("Positive value: " + (double) positive/arr.length );
        System.out.println("Negative value: " +(double) negative/arr.length );
        System.out.println("Neutral value: " + (double) neutral/arr.length );
    }
}

File: src/main/kotlin/plus_minus/plusMinus.kt
package plus_minus

import java.util.*

fun plusMinus() {

    val random = Random()
    val arr = IntArray(5)

    var positive = 0
    var negative = 0
    var neutral = 0

    for (i in arr.indices) {
        arr[i] = random.nextInt()
        println(arr.contentToString())

        when {
            arr[i] > 0 -> {
                positive++
            }
        }
        when {
            arr[i] < 0 -> {
                negative++
            }
        }
        when {
            arr[i] == 0 -> {
                neutral++
            }
        }
    }
    println(positive)
    println(negative)
    println(neutral)

}